routes/register.ts

```diff
@@ -10,18 +11,28 @@ router.post('/', async (_req: Request, res: Response, next: NextFunction): Promi
             username: _req.body.username
         });
 
+
         if (userExists) {
             res.status(409).send({message: 'Account with this username already exists!'});
         }
 
-        const user = await User.create(_req.body);
+        const defaultRole = await Role.findOne({name: 'User'});
 
-        jwt.sign({user}, process.env.SECRET_KEY as string, {expiresIn: '1h'}, (err, token) => {
-            if (err) {
-                res.send({message: 'Token generation failed.'});
-            }
-            res.send({user, token});
-        });
+        if (defaultRole) {
```

Comment:
if statement shouldnt be needed here, as it is a default role it SHOULD always exist